backends/tfhe-cuda-backend/src/cuda_bind.rs

```diff
@@ -311,6 +311,40 @@ extern "C" {
         num_samples: u32,
     );
 
+    /// This scratch function allocates the necessary amount of data on the GPU for
+    /// the public function packing keyswitch implementation on 64-bit
+    pub fn scratch_fp_keyswitch_lwe_list_to_glwe_64(
```

Comment:
it seems none of the extern "C" functions are marked unsafe, is that expected ?

Reply:
I suppose most of them should be unsafe, since most of them are async. I will open an issue to fix that in a new PR.